node_modules/oae-core/members/js/members.js

```diff
@@ -46,7 +46,7 @@ define(['jquery', 'oae.core', 'jquery.history'], function($, oae) {
 
             var url = '/api/group/' + widgetData.context.id + '/members';
             if (query) {
-                url = '/api/search/members/'  + widgetData.context.id;
+                url = '/api/search/group-members-library/'  + widgetData.context.id;
```

Comment:
Special consideration for Georgia Tech would of course be appreciated, but
the project is probably better served by a standard deprecation process
which anyone could follow, to which Georgia Tech could be asked to refer to
along with everyone else. For the moment, Georgia Tech is wearing the hats
of two distinct stakeholders, but in a few months those hats will be on
separate heads again.

I haven't yet looked at other API policies for comparable projects as a
point of comparison, but will do so.

On Friday, March 27, 2015, Branden Visser <notifications@github.com> wrote:

> In node_modules/oae-core/members/js/members.js
> <https://github.com/oaeproject/3akai-ux/pull/3999#discussion_r27331858>:
>
> > @@ -46,7 +46,7 @@ define(['jquery', 'oae.core', 'jquery.history'], function($, oae) {
> >
> >              var url = '/api/group/' + widgetData.context.id + '/members';
> >              if (query) {
> > -                url = '/api/search/members/'  + widgetData.context.id;
> > +                url = '/api/search/group-members-library/'  + widgetData.context.id;
>
> From a product development standpoint we have a general convention that
> breaking changes require at least a minor version bump. So those with their
> own OAE deployments should be wary of API changes both internal and
> external and test accordingly prior to releasing a minor version upgrade.
>
> From a hosting provider stand-point, we do not have a policy on releasing
> non-backwards-compatible API changes to production. It seems we should at
> least get acknowledgement from GeorgiaTech before releasing such a change,
> which is simple enough. I'm not aware of any other production API consumers
> at this point, but we can probably get an idea of external usage from our
> web analytics, actually.
>
> @nicolaasmatthijs <https://github.com/nicolaasmatthijs> do you have any
> idea off hand how much usage comes from what looks like direct API access,
> if any?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oaeproject/3akai-ux/pull/3999/files#r27331858>.
>